src/elements/content-sharing/__tests__/ContentSharing.test.js

```diff
@@ -373,6 +373,33 @@ describe('elements/content-sharing/ContentSharing', () => {
             );
             expect(wrapper.find(UnifiedShareModal).prop('sharedLink')).toEqual(MOCK_SHARED_LINK);
         });
+
+        test('should call share() from onRemoveLink() and remove the existing shared link', async () => {
+            let wrapper;
+            await act(async () => {
+                wrapper = getWrapper({ itemType: TYPE_FILE });
+            });
+            wrapper.update();
+
+            const usm = wrapper.find(UnifiedShareModal);
+            expect(usm.prop('sharedLink')).toEqual(MOCK_SHARED_LINK);
+
+            convertItemResponse.mockReset();
+            convertItemResponse.mockReturnValue(MOCK_CONVERTED_ITEM_DATA_WITHOUT_SHARED_LINK);
+
+            await act(async () => {
+                usm.invoke('onRemoveLink')();
+            });
+            wrapper.update();
+            expect(share).toHaveBeenCalledWith(
+                { id: MOCK_ITEM_ID, permissions: {} },
+                ACCESS_NONE,
+                expect.anything(),
+                expect.anything(),
+                CONTENT_SHARING_SHARED_LINK_UPDATE_PARAMS,
+            );
+            expect(wrapper.find(UnifiedShareModal).prop('sharedLink')).toEqual(MOCK_NULL_SHARED_LINK);
```

Comment:
Can this be simplified to `usm.prop('sharedLink')`?

Reply:
That was my initial thought as well. However, `usm` refers to the `UnifiedShareModal` in the wrapper _before_ the update, so we have to retrieve a new reference here.